manifests/server.pp

```diff
@@ -131,6 +135,7 @@
   validate_string($rootdn)
   validate_string($rootpw)
   validate_absolute_path($directory)
+  validate_string($backend)
```

Comment:
Do we want to add some logic here to verify the backend? i.e. ensure it's bdb, hdb, etc?